Common/Utilities.cs

```diff
@@ -2911,47 +2905,24 @@ public static void UploadFileToFunctionApp(IPublishingProfile profile, string fi
             }
         }
 
-        public static void UploadFilesToContainer(string connectionString, string containerName, params string[] filePaths)
+        public static void UploadFilesToBlob(string connectionString, string containerName, params string[] filePaths)

```

Comment:
Keep original method name.

Reply:
Updated, and these try catch also need to be kept for new client create exception.
Thank you for reviewing.